client/payment-details/utils/tax-descriptions.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Supported tax description codes for different countries and tax types.
+ * These codes are used to identify and localize tax descriptions in the UI.
+ *
+ * Format: [Country Code] [Tax Type]
+ * Examples:
+ * - "US VAT" for United States Value Added Tax
+ * - "JP JCT" for Japan Consumption Tax
+ * - "AU GST" for Australian Goods and Services Tax
+ */
+type TaxDescriptionKey =
+	| 'AT VAT' // Austria VAT
```

Comment:
(minor) do we get any advantage by specifically typing each one of these? I wonder if we can just declare `TaxDescriptionKey` as a `string`, and `taxDescriptions` as `Record< string, string >` 🤷

Reply:
Fair point, I removed the type here [5585b7a](https://github.com/Automattic/woocommerce-payments/pull/10707/commits/5585b7af2fc6dc6b9d2160e602aa1d9f423b974a) thanks!

Reply:
I see the type removed in d6ae22e511c1badc39b2ed828689627aeb1c13b8 , but it looks like it came back later 😬

Reply:
> I see the type removed in https://github.com/Automattic/woocommerce-payments/commit/d6ae22e511c1badc39b2ed828689627aeb1c13b8 , but it looks like it came back later 😬

Hmm not sure if I follow you. I removed the `TaxDescriptionKey` in the commit and changed the type of `taxDescriptions` to `Record< string, string >`. Do you find something missing?

Reply:
Truly not sure - the individual `TaxDescriptionKey` values were showing up in the diff again, but now they're not again 🤷